KlockanAPI/src/KlockanAPI.Infrastructure/Repositories/CourseRepository.cs

```diff
@@ -40,19 +40,23 @@ public async Task<Course> DeleteCourseAsync(Course course)
         return course;
     }
 
-    public async Task<Course?> UpdateCourseAsync(Course course)
+    public async Task<Course> UpdateCourseAsync(Course course)
     {
-        var _course = await _context.Courses.FindAsync(course.Id);
-
-        if (_course != null)
-        {
-            _course.Name = course.Name;
-            _course.Sessions = course.Sessions;
-            _course.SessionDuration = course.SessionDuration;
-            _course.Description = course.Description;
-            await _context.SaveChangesAsync();
-        }
-
-        return _course;
+        // var _course = await _context.Courses.FindAsync(course.Id);
+
+        // if (_course != null)
+        // {
+        //     _course.Name = course.Name;
+        //     _course.Sessions = course.Sessions;
+        //     _course.SessionDuration = course.SessionDuration;
+        //     _course.Description = course.Description;
+        //     await _context.SaveChangesAsync();
+        // }
+
+        // return _course;
```

Comment:
is necesary mantain this code?

Reply:
No, it's not. Just removed the code. Marked as solved.